core/amber/src/test/scala/edu/uci/ics/texera/workflow/operators/intervalJoin/IntervalOpExecSpec.scala

```diff
@@ -92,7 +94,7 @@ class IntervalOpExecSpec extends AnyFlatSpec with BeforeAndAfter {
     for (k <- leftInput.indices) {
       for (i <- rightInput.indices) {
         dataType match {
-          case AttributeType.INTEGER => {
+          case AttributeType.INTEGER =>
```

Comment:
are there any guidelines on whether we should add `{}` around case? I kinda feel adding a bracket looks safer

Reply:
https://stackoverflow.com/questions/23836619/to-brace-or-not-to-brace-case-statement-block there are always debates on those. I was just following IntelliJ recommendations. and the guidance is here https://docs.scala-lang.org/style/control-structures.html#curly-braces